server/templates/choropleth.html

```diff
@@ -0,0 +1,33 @@
+{#
```

Comment:
Didn't this file move to templates/tools/choropleth.html? This looks like an old version of the file, please rm.